libr/core/cmd_eval.c

```diff
@@ -199,7 +223,19 @@ static int cmd_eval(void *data, const char *input) {
 	char *p;
 	RCore *core = (RCore *)data;
 	switch (input[0]) {
-	case 't': // env
+	case '\0': // "e"
+		r_config_list (core->config, NULL, 0);
+		break;
+	case '?': // "e?"
+	default:
+		switch (input[1]) {
+		case '?': r_config_list (core->config, input+2, 2); break;
+		default: r_config_list (core->config, input+1, 2); break;
+		case '\0':
+			r_core_cmd_help (core, help_msg_e);
+		}
```

Comment:
missing break, not necessary but looks more consistent

Reply:
Done